spec/nylas/api_spec.rb

```diff
@@ -56,6 +56,59 @@
     end
   end
 
+  describe "#free_busy" do
+    it "returns `Nylas::FreeBusyCollection` for free busy details" do
+      emails = ["test@example.com", "anothertest@example.com"]
+      start_time = 1_609_439_400
+      end_time = 1_640_975_400
+      client = Nylas::HttpClient.new(
+        app_id: "not-real",
+        app_secret: "also-not-real",
+        access_token: "seriously-unreal"
+      )
+      api = described_class.new(client: client)
+      free_busy_response = [
+        {
+          object: "free_busy",
+          email: "test@example.com",
+          time_slots: [
+            {
+              object: "time_slot",
+              status: "busy",
+              start_time: 1_609_439_400,
+              end_time: 1_640_975_400
+            }
+          ]
+        }
+      ]
+      allow(client).to receive(:execute).with(
+        method: :post,
+        path: "/calendars/free-busy",
```

Comment:
If you ended up making the path a constant, you could refer to the constant directly here, maybe the test is slightly less brittle then?

Reply:
I would keep the real string here in the test but using constant in code d7a0a721b0a6afe0b62a111eaf0692d14d732439